### PR TITLE
Make reference to i18n-glossary informative

### LINKED ISSUE #48: WG Note cannot be listed as normative reference??
Term code point is [defined by referring i18n-glossary](https://github.com/w3c/imsc-hrm/blob/e286e07dc7ff4d0074e898aaf362f2c948d89ced/spec/imsc-hrm.html#L226), which is non-rectrack and is WG Note.
I believe we cannot mark WG Note as [section F.1 normative reference](https://www.w3.org/TR/imsc-hrm/#normative-references).

### PR DESCRIPTION
**Pull request by palemieux**
Closes #48

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/imsc-hrm/pull/62.html" title="Last updated on Dec 15, 2022, 7:15 PM UTC (22ca4ef)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/imsc-hrm/62/f923e85...22ca4ef.html" title="Last updated on Dec 15, 2022, 7:15 PM UTC (22ca4ef)">Diff</a>

### CODE REVIEW

**Review by nigelmegitt — APPROVED** [review]
LGTM

File: spec/imsc-hrm.html [hunk 1]
```diff
@@ -223,9 +223,7 @@ <h2>Terms and Definitions</h2>
       data-cite="ttml2#terms-character-information-item">Character Information Item</dfn>.</p>
 
       <p class="note">The term <a>character</a> is for practical purposes
-        the same as a <a>code point</a>.</p>
-
-      <p><dfn data-lt="code point|code points">code point</dfn>. As defined by [[i18n-glossary]].</p>
+        the same as a <dfn>code point</dfn>, as defined by [[i18n-glossary]].</p>
```
Comment (himorin) [inline spec/imsc-hrm.html:226]: still normatively referenced here? https://respec.org/docs/#normative-vs-informative-references

Comment (palemieux) [reply spec/imsc-hrm.html:226]: _if the reference is in a section marked "informative", or an example, note, or figure, then ReSpec automatically makes the reference non-normative._

We can use the explicit `[[?]]` syntax, which is new.

Comment (himorin) [reply spec/imsc-hrm.html]: Ah, sorry. I've just checked the changed lines only...
Yes, respec will handle reference in note (or any other non-normative part) as informative, and list into informative references section.

**Pull request merged**